lib/osf-components/addon/components/paginated-list/layout/template.hbs

```diff
@@ -1,28 +1,53 @@
 {{else if (or @items.length (and this.loading this.placeholderCount))}}
-    <ul class='list-group m-md'>
-        {{yield (hash
-            header=(component 'paginated-list/x-header')
-        )}}
-        {{#if this.loading}}
-            {{#each (range 0 this.placeholderCount)}}
-                {{yield (hash
-                    item=(component 'paginated-list/x-item')
-                    doReload=(action @doReload)
-                )}}
-            {{/each}}
-        {{else if @items.length}}
-            {{#each @items as |item index|}}
-                {{#unless item.isDeleted}}
-                    {{yield (hash
-                        item=(component 'paginated-list/x-item' item=item index=index)
-                        doReload=(action @doReload)
+    {{#if this.isTable}}
+        <table>
+            {{yield (hash
+                            header=(component 'paginated-list/x-header' isTable=this.isTable)
                     )}}
-                {{/unless}}
-            {{/each}}
-        {{/if}}
-    </ul>
+            <tbody>
+                {{#if this.loading}}
+                    {{#each (range 0 this.placeholderCount)}}
+                        {{yield (hash
+                                        item=(component 'paginated-list/x-item' isTable=this.isTable)
+                                        doReload=(action @doReload)
+                                )}}
+                    {{/each}}
+                {{else if @items.length}}
+                    {{#each @items as |item index|}}
+                        {{#unless item.isDeleted}}
+                            {{yield (hash
+                                            item=(component 'paginated-list/x-item' isTable=this.isTable item=item index=index)
+                                            doReload=(action @doReload)
+                                    )}}
+                        {{/unless}}
+                    {{/each}}
+                {{/if}}
+            </tbody>
+        </table>
+    {{else}}
+        <ul class='list-group m-md'>
+            {{yield (hash header=(component 'paginated-list/x-header'))}}
+            {{#if this.loading}}
+                {{#each (range 0 this.placeholderCount)}}
+                    {{yield (hash
+                                    item=(component 'paginated-list/x-item')
+                                    doReload=(action @doReload)
+                            )}}
+                {{/each}}
+            {{else if @items.length}}
+                {{#each @items as |item index|}}
+                    {{#unless item.isDeleted}}
+                        {{yield (hash
+                                        item=(component 'paginated-list/x-item' item=item index=index)
+                                        doReload=(action @doReload)
+                                )}}
+                    {{/unless}}
+                {{/each}}
+            {{/if}}
+        </ul>
+    {{/if}}
```

Comment:
I'm not crazy about the large amount of duplication here, but also not sure its worth factoring out into a separate component. Even if it is, we'll leave that as an exercise for the future. Could you add a `{{!-- TODO: --}}` comment here indicating that we should probably do that?